setup.py

```diff
@@ -0,0 +1,32 @@
+from pkgutil import walk_packages
+
+from setuptools import setup
+
+import hdmi
+
+
+def _find_packages(path='.', prefix=''):
+
+        yield prefix
+        prefix += "."
+        for _, name, ispkg in walk_packages(path,
+                                            prefix,
+                                            onerror=lambda x: x):
+            if ispkg:
+                yield name
+
+
+def find_packages():
```

Comment:
Setuptools has a `find_packages` function, I have only seen a custom find packages used if `distutils` and `setuptools` support is desired.  Use the setuptools `find_packages` and/or use distutil if setuptools is not available.